src/styles/stars.css

```diff
@@ -0,0 +1,41 @@
+.card__block-stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__block-price {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 96px;
+  height: 16px;
+}
+
+.stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars-star:nth-child(n+5) {
+  background-image: url(/images/star.svg);
```

Comment:
Not fixed, 
![image](https://user-images.githubusercontent.com/54936044/232195411-27d7fd1e-1957-4e2d-a422-b329bf7052a6.png)